lbrynet/file_manager/EncryptedFileDownloader.py

```diff
@@ -35,18 +34,21 @@ class ManagedEncryptedFileDownloader(EncryptedFileSaver):
     STATUS_RUNNING = "running"
     STATUS_STOPPED = "stopped"
     STATUS_FINISHED = "finished"
-
+    """
+    These are started by EncryptedFileManager, aka, file_manager
+    """
     def __init__(self, rowid, stream_hash, peer_finder, rate_limiter,
                  blob_manager, stream_info_manager, lbry_file_manager,
                  payment_rate_manager, wallet, download_directory,
                  file_name=None):
+
         EncryptedFileSaver.__init__(self, stream_hash, peer_finder,
                                     rate_limiter, blob_manager,
                                     stream_info_manager,
                                     payment_rate_manager, wallet,
                                     download_directory,
                                     file_name)
-        self.sd_hash = None
```

Comment:
The attribute should be set here to `None` just so it's not a new attribute being added later on

Reply:
This attribute was moved to the class that it inherits from as part of this work
>
ManagedEncryptedFileDownloader had a function load_file_attribute() which is used to load claim information along with the sd_hash. Since #983 removes claim information from files, now all this function does is load the sd_hash. Move this task to the base class EncryptedFileDownloader in function set_stream_info() where it gets stream information (cleaner since we have less setup functions that needs to be run in order to load file information).